packages/sdk/src/lib.ts

```diff
@@ -95,9 +111,19 @@ export class PhotonClient {
     this.audience = audience;
     this.uri = uri;
     this.clinicalUrl = getClinicalUrl(uri);
+
+    this.clinicalApiUri = `${
+      this.clinicalUrl?.includes('photon')
```

Comment:
this I'm looking for feedback on... on the one hand we could pass the clinical uri the way we're passing lambdas... or we could just derive it the way @marisbest2. ain't pretty but it does the job, wdyt?

Reply:
check again